Robot/src/main/java/com/swrobotics/robot/commands/AutoAimCommand.java

```diff
@@ -0,0 +1,44 @@
+package com.swrobotics.robot.commands;
+
+import com.swrobotics.robot.subsystems.drive.DrivetrainSubsystem;
+import com.swrobotics.robot.subsystems.vision.Limelight;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class AutoAimCommand extends CommandBase {
+    private final Limelight limelight;
+    private final DrivetrainSubsystem drivetrainSubsystem;
+
+    Rotation2d limelight_target_angle;
```

Comment:
Just a style thing - would recommend making these private and use camel case

Reply:
Sorry too much rust